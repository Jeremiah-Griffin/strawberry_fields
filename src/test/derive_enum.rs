use strawberry_fields::StrawberryFields;

#[derive(StrawberryFields)]
enum Dummy {
    First,
    Second,
}
fn main() {}
