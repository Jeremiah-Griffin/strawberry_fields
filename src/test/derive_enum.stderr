error: proc-macro derive panicked
 --> src/test/derive_enum.rs:3:10
  |
3 | #[derive(StrawberryFields)]
  |          ^^^^^^^^^^^^^^^^
  |
  = help: message: StrawberryFields may only be derived for structs
